### PR TITLE
Statistics v2 4

### PR DESCRIPTION
**Pull request by kapapp**
Statistics page redone (version for grandparents), added a spinner, changed color for stat/profile

### CODE REVIEW

File: app/src/main/res/drawable-v24/oval.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
     <item>
         <shape android:shape="oval">
-            <solid android:color="@color/black"/>
+            <solid android:color="@color/green_blue_crayola"/>
```
Comment (CHOCOLVT) [inline app/src/main/res/drawable-v24/oval.xml:6]: magnifique

**Review by JorisMonnet — APPROVED** [review]
Some dependencies need to be updated
Your layouts are no longer fixed with magic number -> good !!
For spinner_item, use gravity start instead of gravity left

File: app/src/main/java/ch/epfl/sdp/blindwar/ui/StatisticsActivity.kt [hunk 2]
```diff
@@ -3,18 +3,53 @@ package ch.epfl.sdp.blindwar.ui
...
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_statistics)
-    }
+        // access the items of the list
+        val modes = resources.getStringArray(R.array.modes_array)
 
+        // access the spinner
+        val spinner = findViewById<Spinner>(R.id.modes_spinner)
```
Comment (CHOCOLVT) [inline app/src/main/java/ch/epfl/sdp/blindwar/ui/StatisticsActivity.kt:30]: nice feature here

**Review by CHOCOLVT — APPROVED** [review]
great refresh of UI

**Pull request merged**